src/System.Memory/tests/Performance/Perf.Span.Contains.cs

```diff
@@ -187,7 +187,7 @@ private static string GenerateInputString(char source, int count, char replaceCh
         {
             new object[] { "string1", '1' },
             new object[] { "foobardzsdzs", 'z' },
-            new object[] { "StrIng", "I" },
+            new object[] { "StrIng", 'I' },
```

Comment:
@grant-d, this test case is redundant and should be removed (the likely culprit of the issue [here](https://github.com/dotnet/corefx/pull/32293#issuecomment-423004577)):
https://github.com/dotnet/corefx/pull/32362/files#diff-0a617e172329d07b60ee8211639d4899R199

```C#
new object[] { GenerateInputString('A', 100, 'X', 70), 'x' },
```

In the case of IndexOf (where the test cases were borrowed from) it wasn't redundant since we had a different StringComparison:
https://github.com/dotnet/corefx/blob/296373a73766e3b409c91e480c8e73aed39ed4a8/src/System.Memory/tests/Performance/Perf.Span.IndexOf.cs#L192

For good measure, I would also change the `replaceChar` character to be different for each test case (we have 'X' in one, change the [other](https://github.com/dotnet/corefx/pull/32362/files#diff-0a617e172329d07b60ee8211639d4899R200) one to be 'Y').

Otherwise, we seem to end up getting duplicate test iterations (Iterations should be 21, not 87).

```text
   System.Memory.Tests.Perf_Span_Contains.ContainsChar_StringIndexOf(input: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"..., value: 'x')                                               | Duration | msec |     21     |    0.309 |      0.002 |    0.308 |    0.318
   System.Memory.Tests.Perf_Span_Contains.ContainsChar_StringIndexOf(input: "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"..., value: 'x')                                               | Duration | msec |     87     |    0.882 |      0.003 |    0.878 |    0.898
   System.Memory.Tests.Perf_Span_Contains.ContainsChar_StringIndexOf(input: "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"..., value: 'x')                                               | Duration | msec |     87     |    0.882 |      0.003 |    0.878 |    0.898
   System.Memory.Tests.Perf_Span_Contains.ContainsChar_StringIndexOf(input: "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"..., value: 'X')                                               | Duration | msec |     21     |    0.309 |      0.002 |    0.308 |    0.317
```

While you are it, might as well change it in other places as well, to avoid confusion:
https://github.com/dotnet/corefx/blob/296373a73766e3b409c91e480c8e73aed39ed4a8/src/System.Memory/tests/Performance/Perf.Span.IndexOf.cs#L193

https://github.com/dotnet/corefx/blob/f641a94ce4b31e01666ff6585adc42afcb925693/src/System.Globalization/tests/Performance/Perf.CompareInfo.cs#L68

Reply:
OK, I understand what happened now. Changes pushed.

Reply:
I chose to replace 'X' and 'Y' with visually-different characters. X & Y look similar.